fbw-common/src/systems/instruments/src/EFB/Settings/sync.ts

```diff
@@ -50,12 +50,12 @@ const settingsToSync: Map<string, SimVar> = new Map([
 
 const settingEnumToSync: Map<string, SimVarEnum> = new Map([
     ['CONFIG_BOARDING_RATE',
-        ['L:A32NX_BOARDING_RATE', 'number', '0',
+        ['L:A32NX_BOARDING_RATE', 'number', 'REAL',
```

Comment:
This does look pretty odd (`number`, `'REAL')  but maybe a subject for a refactor PR not a fix one.

Reply:
I think you designed it? :D